examples/iris-example/go.mod

```
@@ -0,0 +1,54 @@
+module example.com/iris
+
+go 1.20
+
+require (
+	github.com/auth0/go-jwt-middleware/v2 v2.1.0
+	github.com/kataras/iris/v12 v12.2.0
+)
+
```

I think similar to the other examples we should add a replace here so the local version of this repo gets used


```suggestion

replace github.com/auth0/go-jwt-middleware/v2 => ./../../

```